-----------------------------------------------------
Manifest: manifests\b\Bluefish\Bluefish\2.2.12rc2
-----------------------------------------------------
URI: http://bluefish.openoffice.nl/index.html, Validation result: Passed
URI: https://bfwiki.tellefsen.net/index.php/Manual_2_ToC, Validation result: Passed
URI: http://bluefish.openoffice.nl/features.html, Validation result: Passed
URI: http://bluefish.svn.sourceforge.net/viewvc/bluefish/trunk/bluefish/COPYING, Validation result: Failed, Http status code: NotFound 
URI: https://www.bennewitz.com/bluefish/stable/binaries/win32/Bluefish-2.2.12rc2-setup.exe, Validation result: Passed

****************************************************************************************
Manifest Content Validation
****************************************************************************************
Status: Completed
##[error] Result: Failed

--- a/manifests/b/Bluefish/Bluefish/2.2.12rc2/Bluefish.Bluefish.installer.yaml
+++ b/manifests/b/Bluefish/Bluefish/2.2.12rc2/Bluefish.Bluefish.installer.yaml
@@ -12,7 +12,7 @@ Installers:
 - InstallerLocale: en-US
   Architecture: x86
   InstallerType: nullsoft
-#  Scope: 
+  Scope: machine
   InstallerUrl: https://www.bennewitz.com/bluefish/stable/binaries/win32/Bluefish-2.2.12rc2-setup.exe
   InstallerSha256: E02D0AE7B613914392A44C50A21002A06EE314FC6DF711662E16EBACDBA44913
   UpgradeBehavior: install

--- a/manifests/b/Bluefish/Bluefish/2.2.12rc2/Bluefish.Bluefish.locale.en-US.yaml
+++ b/manifests/b/Bluefish/Bluefish/2.2.12rc2/Bluefish.Bluefish.locale.en-US.yaml
@@ -12,7 +12,7 @@ PublisherSupportUrl: https://bfwiki.tellefsen.net/index.php/Manual_2_ToC
 PackageName: Bluefish
 PackageUrl: http://bluefish.openoffice.nl/features.html
 License: GNU GPL
-LicenseUrl: http://bluefish.svn.sourceforge.net/viewvc/bluefish/trunk/bluefish/COPYING
+LicenseUrl: https://sourceforge.net/p/bluefish/code/HEAD/tree/trunk/bluefish/COPYING
 #Copyright: 
 #CopyrightUrl: 
 ShortDescription: Bluefish is a powerful editor targeted towards programmers and webdevelopers, with many options to write websites, scripts and programming code.

